rotkehlchen/externalapis/cryptocompare.py

```diff
@@ -651,6 +702,15 @@ def query_historical_price(
         - RemoteError if there is a problem reaching the cryptocompare server
         or with reading the response returned by the server
         """
+        # NB: check if the from..to asset price (or viceversa) is a special
+        # histohour API case.
+        price = self._check_and_get_special_histohour_price(
```

Comment:
Shouldn't this special case only fire if cryptocompare actually returns price(0)? That's the only time we need to check this case.

Reply:
As discussed in discord with the added test to warn us this is okay.